src/plugins/experimental-discord/createGraph.js

```diff
@@ -212,18 +219,28 @@ export function createGraph(
       const reactions = repo.reactions(channel.id, message.id);
       for (const reaction of reactions) {
         const emojiRef = Model.emojiToRef(reaction.emoji);
-        const reactionWeight = emojiWeights[emojiRef];
+        let reactionWeight = emojiWeights[emojiRef];
+        if (reactionWeight === null || reactionWeight === undefined) {
+          reactionWeight = 1;
+        }
 
         const reactingMember = memberMap.get(reaction.authorId);
         if (!reactingMember) {
           // Probably this user left the server.
           continue;
         }
 
-        const node = reactionNode(reaction, message.timestampMs, guild);
-        if (reactionWeight != null) {
-          wg.weights.nodeWeights.set(node.address, reactionWeight);
+        // get the weight of the highest weight role the reacting user has
+        let roleWeight = roleWeightConfig.defaultWeight;
+        const roleWeights = roleWeightConfig.roleWeights;
+        for (const roleRef of reactingMember.roles) {
+          if (roleWeights[roleRef] > roleWeight) {
+            roleWeight = roleWeights[roleRef];
+          }
```

Comment:
`roleWeights[roleRef]` may be undefined, so we're doing `undefined > number` comparisons.  this works but it's sketchy. let's do something like:

```js
const matchingWeight = roleWeights[roleRef];
if (matchingWeight != null && matchingWeight > roleWeight) {
  roleWeight = matchingWeight;
}
```